src/v2/Components/Artwork/GridItem.tsx

```diff
@@ -125,7 +122,7 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
         style={style}
       >
         <Placeholder style={{ paddingBottom: artwork.image.placeholder }}>
-          <RouterLink
+          <Link
```

Comment:
@dzucconi - for future grep / refactorability we should keep router-related links named `RouterLink` or at least encode that string in some larger name -- that is, until we rename everything all at once to `Link`.